corehq/apps/app_manager/management/commands/find_broken_builds.py

```diff
@@ -144,3 +157,8 @@ def find_broken_builds(checker, builds_ids):
                 build.get('copy_of'),
                 error,
             )
+            if reason:
+                build = Application.wrap(build)
+                build.build_broken = True
+                build.build_broken_reason = reason
```

Comment:
nitpick: Should the [comment](https://github.com/dimagi/commcare-hq/blob/6c3d20ca9dc4d3cee9d7bcffc4b7afd487d8333a/corehq/apps/app_manager/models.py#L4245-L4248) related to build_broken_reason be updated to reflect this usage?

Reply:
yes, updated